ci_framework/roles/hive/tasks/openstack_claim_cluster.yml

```diff
@@ -0,0 +1,75 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# Claims a cluster from a the provided pool
+- name: Claim an OpenShift Cluster
+  vars:
+    claim_name: "{{ cifmw_hive_openstack_claim_name }}"
+    cluster_pool: "{{ cifmw_hive_openstack_pool_name }}"
+    rtn_kube: "{{ cifmw_hive_artifacts_dir }}"
+    life_time: "{{ cifmw_hive_lifetime | default(false) }}"
+    namespace: "{{ cifmw_hive_namespace }}"
+
+  block:
+    - name: Check if ClusterPool exists
+      when: not (cifmw_hive_dry_run | default(false)) | bool
+      environment:
+        PATH: "{{ cifmw_path }}"
+        KUBECONFIG: "{{ kubeconfig | default(cifmw_hive_kubeconfig) }}"
```

Comment:
Where is that kubeconfig defined? I was assuming that this role is gonna use always the one pointed by `cifmw_hive_kubeconfig`. If it's an optional variable that may be passed from the scenario files or from wherever source the name is misleading, as no one knows if this kubeconfig refers to the one of the target cluster for OSP deployments or to the hive one.

Reply:
@psathyan are we planning to override cifmw_hive_kubeconfig with kubeconfig somewhere else?

Reply:
ack, removed kubeconfig var